parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -330,6 +338,30 @@ pub mod pallet {
 				.saturating_add(len_fee)
 				.saturating_add(T::PricingParameters::get().rewards.local)
 		}
+
+		/// Burn the amount of the fee embedded into the XCM for teleports
+		pub fn burn_fees(para_id: ParaId, fee: BalanceOf<T>) -> DispatchResult {
+			let dummy_context =
+				XcmContext { origin: None, message_id: Default::default(), topic: None };
+			let dest = MultiLocation { parents: 1, interior: X1(Parachain(para_id.into())) };
+			let fees = (MultiLocation::parent(), fee.saturated_into::<u128>()).into();
```

Comment:
Can this be inferred from the `T::Token`?

Reply:
Actually, this PR makes it not necessary to have a `T::Token` config field. Can we remove it @ron?

Reply:
Still in use to reward relayer
https://github.com/Snowfork/snowbridge/blob/f7bd88475c9f52ce19361018de9514b7d0d0979a/parachain/pallets/inbound-queue/src/lib.rs#L270 so I prefer to keep it.